t/29-request_logger_test.t

```diff
@@ -0,0 +1,27 @@
+use Test::More;
+use Mojo::WebSocketProxy::RequestLogger;
+
+my $context = {'correlation_id' => '1234'};
+my $logger = Mojo::WebSocketProxy::RequestLogger->new(context => $context);
+
+
+subtest 'Test log_message method' => sub {
+    $logger->info('This is a info message');
+    $logger->warn('This is a warning message');
+    $logger->trace('This is a trace message');
+    $logger->error('This is an error message');
+    $logger->debug('This is debug message');
+    $logger->infof('This is a info message %s', ['abc', '221']);
+    $logger->warnf('%s This is a warning message ', {with => 'params'});
+    $logger->tracef('This is a trace %s message', "with params");
+    $logger->errorf('This is an error %s message',"with params");
+    $logger->debugf('This is debug message with %s', 'params');
+    pass('All log levels tested');
```

Comment:
how do we test it. just make sure that we implemented these methods?

Reply:
we are checking the implementation for warn message in another test https://github.com/binary-com/perl-Log-Any-Adapter-DERIV/pull/20/files#diff-5ce3bf7ac545da06738a7184a78470998388c0ec9e97849cc9b67b0dce1a826a, all other are being put here to check methods are implemented